src/shortcuts/Shortcut.jsx

```diff
@@ -165,7 +165,7 @@ class Shortcut extends Context {
     }
 
     setAttributeIsSetByLabel(name, val) {
-        if (this.hasParentContext()) this.parentContext.setAttributeIsSetByLabel(name, val);
+        if (this.hasParentContext() && this.parentContext) this.parentContext.setAttributeIsSetByLabel(name, val);

```

Comment:
Shouldn't this.hasParentContext only be true when we have a parent context? Do we know when this was causing issues in particular?